FridgeExercise/Refrigerator.cs

```diff
@@ -0,0 +1,249 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+using System.Xml.Linq;
+
+namespace FridgeExercise
+{
+	internal class Refrigerator
+	{
+		public int RefrigeratorID { get; set; }
+		private int CounterID = 1;
+		public string Model { get; set; }
+		public string Color { get; set; }
+		public int NumberOfShelves { get; set; }
+		public List<Shelf> Shelves { get; set; }
+
+
+		public Refrigerator(string model, string color, int numberOfShelves) {
```

Comment:
```suggestion
		public Refrigerator(string model, string color, int numberOfShelves)
		{
```

Styling - make sure to keep consistency with the brackets